support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -95,6 +95,14 @@ export type RegularPaymentRequestAddress = {|
   city: Option<string>,
 |};
 
+type GiftRecipientType = {|
+  title?: Option<Title>,
+  firstName: string,
+  lastName: string,
+  email?: Option<string>,
+  // message: Option<string>,
```

Comment:
this will conflict with https://github.com/guardian/support-frontend/pull/2702/files#r494923958
the format the JSON should arrive in for giftee details is changed in my proposal, so we will need to chat some time!
@lucymonie

Reply:
👍